webservices/schemas.py

```diff
@@ -584,7 +584,7 @@ class CandidateHistoryTotalSchema(schemas['CandidateHistorySchema'],
         'recipient_name': ma.fields.Str(),
     },
     options={
-        'exclude': ('committee', 'recipient')
+        'exclude': ('idx', 'committee', 'recipient')
     },
 )
 
```

Comment:
Will excluding `idx` from api returns break someone’s app or script using the current endpoint?
@lbeaufort @fec-jli